android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -293,10 +295,16 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull Result result) {
         }
     }
 
-    private void setupPurchases(String apiKey, String appUserID, @Nullable Boolean observerMode, final Result result) {
+    private void setupPurchases(String apiKey, String appUserID, @Nullable Boolean observerMode,
+                                @Nullable Boolean useAmazon, final Result result) {
         if (this.applicationContext != null) {
             PlatformInfo platformInfo = new PlatformInfo(PLATFORM_NAME, PLUGIN_VERSION);
-            CommonKt.configure(this.applicationContext, apiKey, appUserID, observerMode, platformInfo);
+            Store store = Store.PLAY_STORE;
+            if (useAmazon != null && useAmazon) {
+                store = Store.AMAZON;
+            }
```

Comment:
does something like this work? 
```suggestion
            Store store = useAmazon == true? Store.AMAZON: Store.PLAY_STORE
```

Reply:
In Kotlin yes, but not in Java. useAmazon is a Boolean, and in order to compare to true, it needs to be unboxed, which can cause a NullPointerException if it's null